lib/dal/src/pkg.rs

```diff
@@ -252,20 +261,26 @@ where
     Key: Eq + PartialEq + std::hash::Hash,
 {
     pub fn new() -> Self {
-        let change_set_map = HashMap::new();
-
         Self {
-            inner: change_set_map,
+            inner: HashMap::new(),
+            overrides: HashMap::new(),
         }
     }
 
     pub fn get(&self, key: &Key) -> Option<&Thing> {
-        self.inner.get(key)
+        match self.overrides.get(key) {
+            Some(override_key) => self.inner.get(override_key),
+            None => self.inner.get(key),
+        }
     }
 
     pub fn insert(&mut self, key: Key, thing: Thing) -> Option<Thing> {
         self.inner.insert(key, thing)
     }
+
+    pub fn insert_override(&mut self, new: Key, old: Key) {
+        self.overrides.insert(new, old);
+    }
```

Comment:
Very subtle, but very import. This ensures we are importing/exporting the right intrinsics. Without this, new systems will never migrate to the new intrinsics (also due to the unique func name assumptions, but that's a long story).